waveform_collection/server.py

```diff
@@ -74,26 +74,31 @@ def gather_waveforms(source, network, station, location, channel, starttime,
             ``http://10.30.5.10:8080`` if using VPN)
         watc_username (str): Username for WATC FDSN server
         watc_password (str): Password for WATC FDSN server
+        verbose (bool): If `False`, all print statements will be blocked. 
+            Default is `True`.
 
     Returns:
         :class:`~obspy.core.stream.Stream` containing gathered waveforms. If
         `return_failed_stations` is `True`, additionally returns a list
         containing station codes that were requested but not downloaded
     """
-
+    
+    # Set up log function if a verbose output is desired
```

Comment:
This is set up regardless of whether or not verbose is desired — suggest this explanatory comment instead
```suggestion
    # log() does nothing if `verbose=False`
```